src/core_plugins/kibana/public/management/sections/objects/change_index_modal.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { groupBy, mapValues, take, get } from 'lodash';
+
+import {
+  KuiModal,
+  KuiModalHeader,
+  KuiModalHeaderTitle,
+  KuiModalBody,
+  KuiModalBodyText,
+  KuiModalFooter,
+  KuiButton,
+  KuiModalOverlay,
+  KuiTable,
+  KuiTableBody,
+  KuiTableHeader,
+  KuiTableHeaderCell,
+  KuiTableRow,
+  KuiTableRowCell,
+  KuiControlledTable,
+  KuiToolBar,
+  KuiToolBarSection,
+  KuiPager,
+} from 'ui_framework/components';
+
+import { keyCodes } from 'ui_framework/services';
```

Comment:
Looks like this isn't exported in the index though, though you could export `ESC_KEY_CODE` from `ui_framework/services/key_codes`

Reply:
https://github.com/elastic/kibana/pull/13663 was never backported to 6.0

Reply:
Nice catch!